src/confluent_kafka/src/Admin.c

```diff
@@ -2835,6 +3195,89 @@ static PyObject *Admin_c_DescribeConsumerGroupsResults_to_py(
         return NULL;
 }
 
+static PyObject *Admin_c_ScramMechanism_to_py(rd_kafka_ScramMechanism_t mechanism){
+        PyObject *result = NULL, *value_dict = NULL;
+        PyObject *args = NULL;
+        value_dict = PyDict_New();
+        cfl_PyDict_SetInt(value_dict, "value",(int) mechanism);
+        args = PyTuple_New(0);
+        result = PyObject_Call(cfl_PyObject_lookup("confluent_kafka.admin",
+                                                   "ScramMechanism"), args, value_dict);
+        Py_DECREF(value_dict);
+        return result;
+}
+
+static PyObject *Admin_c_ScramCredentialInfo_to_py(const rd_kafka_ScramCredentialInfo_t *scram_credential_info){
+        PyObject *result = NULL;
+        PyObject *args = NULL;
+        result = PyDict_New();
+        PyDict_SetItemString(result,"mechanism",Admin_c_ScramMechanism_to_py(rd_kafka_ScramCredentialInfo_mechanism(scram_credential_info)));
```

Comment:
Calling multiple nested functions decrease readability. Create local variables and use them.